hoot-services/src/main/java/hoot/services/controllers/ingest/CustomScriptResource.java

```diff
@@ -297,6 +282,21 @@ public Response getScriptsList(@Context HttpServletRequest request) {
                     String content = FileUtils.readFileToString(translationFile, "UTF-8");
                     JSONObject oScript = getScriptObject(content);
 
+                    JSONObject json = new JSONObject();
+                    json.put("id", translationRecord.get(translations.id));
+                    json.put("name", translationName);
+                    json.put("userId", translationRecord.get(translations.userId));
+                    json.put("date", translationRecord.get(translations.createdAt).toString());
+                    json.put("folderId", folderId);
+                    json.put("displayPath", translationPath);
```

Comment:
this was the only part that i actually changed. uses the logic from like 278 to check if there is a parent folder path or not

Reply:
:sunglasses: :dog: